source/tests/mocks/ODClientTest.cpp

```diff
@@ -410,7 +410,33 @@ bool ODClientTest::processMessage(ServerNotificationType cmd, ODPacket& packetRe
                 BOOST_CHECK(packetReceived >> walkDirection);
```

Comment:
Again I can't find how to comment on a single commit, GH is failing me :°)

Just wanted to say that you should remember to make commit logs as "short title + newline + commit description", instead of a one liner as shown above. So this log should have been something like:
```
Pass information about the played animation in unit tests messages

In the unit tests, we should call a function telling the animation played by entities for
every message that can change the animation as it is heavily used to know what's happening.
```
(the newline is important to separate the first "commit title" from the "commit description").